### ISSUE TITLE
Question: Narrowing scope for feed to just a single blog

### ISSUE THREAD

**Issue by DarkNeedle** [akash3444/hashnode-clone#1]
Question I have, I like the way this was constructed, very clean. What would be the necessary changes to narrow the scope of the feed to just a single blog. Almost like a pure blog template? I assume the GraphQL query would have to be extended in queries.ts and adjustment to the API in feeds to add a getPostList or such. Or is there a better way to just modify the Feed query/api to just pull back a single blog feed. using the example of engineering. hashnode.com

thanks
DN

**Comment by akash3444** [comment]
@DarkNeedle Yeah, you could definitely narrow the scope of the feed to just a single blog. However, I suggest using [Hashnode Starter Kit](https://github.com/Hashnode/starter-kit) for ease of access and additional benefits.
